test/modules/aws-ec2-integration.ts

```diff
@@ -3,10 +3,9 @@ import * as iasql from '../../src/services/iasql'
 import { getPrefix, runQuery, runInstall, runUninstall, runApply, finish, execComposeUp, execComposeDown, runSync, } from '../helpers'
 
 const dbAlias = 'ec2test';
-// specific to us-west-2, varies per region
-const region = 'us-west-2'
-const amznAmiId = 'ami-06cffe063efe892ad';
-const ubuntuAmiId = 'ami-0892d3c7ee96c0bf7';
+const region = process.env.AWS_REGION;
+const amznAmiId = 'resolve:ssm:/aws/service/ami-amazon-linux-latest/amzn2-ami-hvm-x86_64-gp2';
+const ubuntuAmiId = 'resolve:ssm:/aws/service/canonical/ubuntu/server/20.04/stable/current/amd64/hvm/ebs-gp2/ami-id';
```

Comment:
wait, this works across regions!?! that is amazing!

Reply:
we should update the docs

Reply:
yes!

Reply:
https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/finding-an-ami.html